mmdet/ops/dcn/functions/deform_conv.py

```diff
@@ -0,0 +1,167 @@
+import torch
+from torch.autograd import Function
+from torch.nn.modules.utils import _pair
+
+from .. import deform_conv_cuda
+
+
+class DeformConvFunction(Function):
+
+    @staticmethod
+    def forward(ctx,
+                input,
+                offset,
+                weight,
+                stride=1,
+                padding=0,
+                dilation=1,
+                deformable_groups=1,
+                im2col_step=64):
```

Comment:
The `bias` argument can be also added for convenience. An assertion can be used to ensure it is False.